### PR TITLE
Cleanup/Modernize

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/scope/no-block-scope.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/scope/no-block-scope.md?ref=a3b3225f9ffc31fdb6941f9a54e3e60320508abe)

File: javascript/core/scope/no-block-scope.md [hunk 1]
```diff
@@ -36,54 +21,85 @@ aspects:
 ---
 ## Content
 
-Functions have their own scopes, however most other blocks do not.
+In JavaScript, every function creates a scope. 
+
+Nested functions create nested scopes. 
+
+When JavaScript tries to figure out what value a particular variable contains, it checks the closest scope. If it doesn't find it there, it keeps going up the scope until it does (or reaches the global scope without finding the value at all).
```
Comment (nem035) [inline javascript/core/scope/no-block-scope.md]:
```suggestion
When JavaScript tries to figure out what value a particular variable contains, it checks the closest scope. If it doesn't find it there, it keeps going up the chain of scopes until it does (or reaches the global scope without finding the value at all).
```

File: javascript/core/scope/no-block-scope.md [hunk 2]
```diff
@@ -31,59 +16,90 @@ aspects:
...
-# No block scope
+# Function vs Block Scope
 
 ---
 ## Content
 
-Functions have their own scopes, however most other blocks do not.
+In JavaScript, every function creates a scope. 
+
+Nested functions create nested scopes.
```
Comment (kapnobatai137) [inline javascript/core/scope/no-block-scope.md]:
```suggestion
In JavaScript, every function creates a scope. Following this logic, nested functions will create nested scopes. 
```

File: javascript/core/scope/no-block-scope.md [hunk 3]
```diff
@@ -31,59 +16,90 @@ aspects:
...
-  var a = 2;
-  alert(a); //2
+
+function two() {
+  let a = 2;
+  alert(a); // 2
 }
+
+global();
+local();
```
Comment (kapnobatai137) [inline javascript/core/scope/no-block-scope.md]: update function calls to their new names
```suggestion
one();
two();
```

**Pull request merged**